src/main/java/use_case/list/moveMedia/MoveInputData.java

```diff
@@ -0,0 +1,34 @@
+package use_case.list.moveMedia;
+
+/**
+ * Input data for the move media use case.
+ */
+public class MoveInputData {
+    private String sourceCollectionName;
+    private String targetCollectionName;
+    private String movieName;
+
+    public String getSourceCollectionName() {
+        return sourceCollectionName;
+    }
+
+    public void setSourceCollectionName(String sourceCollectionName) {
+        this.sourceCollectionName = sourceCollectionName;
+    }
+
+    public String getTargetCollectionName() {
+        return targetCollectionName;
+    }
+
+    public void setTargetCollectionName(String targetCollectionName) {
+        this.targetCollectionName = targetCollectionName;
+    }
+
+    public String getMovieName() {
+        return movieName;
+    }
+
+    public void setMovieName(String movieName) {
+        this.movieName = movieName;
```

Comment:
We really don't have the application set up for anything other than movies right now, but the hope would be to handle more types of media, and I think the method names should reflect that.

```suggestion
    private String mediaName;

    public String getSourceCollectionName() {
        return sourceCollectionName;
    }

    public void setSourceCollectionName(String sourceCollectionName) {
        this.sourceCollectionName = sourceCollectionName;
    }

    public String getTargetCollectionName() {
        return targetCollectionName;
    }

    public void setTargetCollectionName(String targetCollectionName) {
        this.targetCollectionName = targetCollectionName;
    }

    public String getMediaName() {
        return mediaName;
    }

    public void setMediaName(String mediaName) {
        this.mediaName = mediaName;
```